pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -98,7 +110,15 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 					attr.capability.Memory = math.MaxFloat64
 				}
 			}
-
+			if len(queue.Queue.Spec.Guarantee.Resource) != 0 {
+				attr.guarantee = api.NewResource(queue.Queue.Spec.Guarantee.Resource)
+			}
+			realCapability := pp.totalResource.Clone().Sub(pp.totalGuarantee.Clone().Sub(attr.guarantee))
```

Comment:
should be "realCapability := pp.totalResource.Clone().Sub(pp.totalGuarantee.Clone().Add(attr.guarantee)) " ? As the totalGuarantee has contains the guarantee of current queue.